app/build.gradle.kts

```diff
@@ -116,7 +116,9 @@ dependencies {
         exclude(group = "org.apache.httpcomponents", module = "httpclient")
     }
     implementation(Dependencies.ThirdParty.jsch)
-    implementation(Dependencies.ThirdParty.sshj)
+    implementation(files("../libs/sshj-0.29.1-SNAPSHOT.jar"))
+    implementation("org.bouncycastle:bcpkix-jdk15on:1.65")
+    implementation("net.i2p.crypto:eddsa:0.3.0")
```

Comment:
These need to go to `buildSrc` eventually

Reply:
Those will go for good once we are no longer including sshj as a JAR, but as a Maven dependency.